### PR TITLE
(QENG-2198) Convert PuppetDB acceptance over to vmpooler.

### PR DESCRIPTION
**Pull request by waynr**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -60,6 +60,9 @@ group :acceptance do
     # use the pinned version
     gem 'beaker', '~>2.11'
   end
+  if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
```
Comment (kbarber) [inline Gemfile]: Can we comment as to why this is needed? It won't make sense to us later on.

File: ext/jenkins/beaker-tests.sh [hunk 2]
```diff
@@ -20,12 +22,24 @@ export BEAKER_project=PuppetDB
 export BEAKER_department=eso-dept
 export BEAKER_PRESERVE_HOSTS=onfail
 
-# S3 params
-export PUPPETDB_PACKAGE_REPO_HOST="puppetdb-prerelease.s3.amazonaws.com"
-export PUPPETDB_PACKAGE_REPO_URL="http://puppetdb-prerelease.s3.amazonaws.com/puppetdb/${PACKAGE_BUILD_VERSION}"
-export PUPPETDB_PACKAGE_BUILD_VERSION=$PACKAGE_BUILD_VERSION
-export BEAKER_CONFIG=$LAYOUT
+# Transform LAYOUT from old static host config style to genconfig2 style
```
Comment (kbarber) [inline ext/jenkins/beaker-tests.sh]: What necessary PR's?

**Comment by kbarber** [comment]
@waynr seems alright, lets see how it passes.

File: Gemfile (hunk 1 above)
Comment (waynr) [inline Gemfile]: I added a comment below this line indicating why it's necessary.

File: ext/jenkins/beaker-tests.sh (hunk 2 above)
Comment (waynr) [inline ext/jenkins/beaker-tests.sh]: I added information about the necessary PRs below this line.

**Comment by waynr** [comment]
@kbarber looks like it passed without a problem, the current test run was triggered by the PR update when I pushed a commit with the comments you requested.

**Comment by kbarber** [comment]
Looks like the completion of this is open-ended in regards to switching to vmpooler, so I'm going to close this PR for now.

**Pull request closed without merging**